### PR TITLE
Replace ETCD with Bolt in ipam plugin

### PR DESCRIPTION
**Pull request by deathbeam**
Instead of using ETCD as k/v store in ipam plugin, use newly introduced
Bolt.

Signed-off-by: Tomas Slusny <slusnucky@gmail.com>

### CODE REVIEW

**Comment by deathbeam** [comment]
```
53 passed, 12 skipped in 723.57 seconds
```

:tada:

File: plugins/contiv/plugin_impl_contiv.go [hunk 1]
```diff
@@ -134,7 +136,7 @@ type InterfaceWithIP struct {
 
 // Init initializes the Contiv plugin. Called automatically by plugin infra upon contiv-agent startup.
 func (plugin *Plugin) Init() error {
-	broker := plugin.ETCD.NewBroker(plugin.ServiceLabel.GetAgentPrefix())
+	broker := plugin.Bolt.NewBroker(plugin.ServiceLabel.GetAgentPrefix())
 	// init map with configured containers
 	plugin.configuredContainers = containeridx.NewConfigIndex(plugin.Log, "containers", broker)
```
Comment (lukasmacko) [inline plugins/contiv/plugin_impl_contiv.go]: Let's keep containeridx in etcd

File: k8s/contiv-vpp.yaml [hunk 2]
```diff
@@ -331,6 +335,10 @@ spec:
                   fieldPath: spec.nodeName
             - name: ETCD_CONFIG
               value: "/etc/etcd/etcd.conf"
+            - name: BOLT_CONFIG
+              value: "/etc/agent/bolt.conf"
+            - name: DEBUG_BOLT_CLIENT
+              value: "true"
```
Comment (lukasmacko) [inline k8s/contiv-vpp.yaml]: Do not enable debug by default

**Comment by lukasmacko** [comment]
Bump version of cn-infra in vpp-agent branch https://github.com/ligato/vpp-agent/tree/contiv-1807 in order to include bolt lib and use the latest commit from the branch in your PR commit Gopkg files as well

**Comment by deathbeam** [comment]
Updated

**Pull request merged**